fragment.go

```diff
@@ -1323,73 +1323,67 @@ func (f *fragment) mergeBlock(id int, data []pairSet) (sets, clears []pairSet, e
 }
 
 // bulkImport bulk imports a set of bits and then snapshots the storage.
-// This does not affect the fragment's cache.
+// The cache is updated to reflect the new data.
 func (f *fragment) bulkImport(rowIDs, columnIDs []uint64) error {
-	f.mu.Lock()
-	defer f.mu.Unlock()
 	// Verify that there are an equal number of row ids and column ids.
 	if len(rowIDs) != len(columnIDs) {
 		return fmt.Errorf("mismatch of row/column len: %d != %d", len(rowIDs), len(columnIDs))
 	}
 
 	// Disconnect op writer so we don't append updates.
-	f.storage.OpWriter = nil
-
+	localBitmap := roaring.NewBitmap()
```

Comment:
Continuing the previously resolved converstion, I think it would be helpful to add a comment above the declaration of `localBitmap` that describes what is happening. Then the "disconnect op writer..." comment should be moved down to be above the "... = nil" line.